app/controllers/workflows_controller.rb

```diff
@@ -1,15 +1,115 @@
 # frozen_string_literal: true
 
 class WorkflowsController < Sellers::BaseController
-  before_action :set_body_id_as_app
+  before_action :set_workflow, only: %i[edit update destroy]
+  before_action :authorize_workflow, only: %i[edit update destroy]
+
+  layout "inertia"
 
   def index
     authorize Workflow
     create_user_event("workflows_view")
+
+    workflows_presenter = WorkflowsPresenter.new(seller: current_seller)
+    render inertia: "Workflows/Index", props: workflows_presenter.workflows_props
+  end
+
+  def new
+    authorize Workflow
+
+    workflow_presenter = WorkflowPresenter.new(seller: current_seller)
+    render inertia: "Workflows/New", props: {
+      context: -> { workflow_presenter.workflow_form_context_props },
+    }
+  end
+
+  def edit
+    workflow_presenter = WorkflowPresenter.new(seller: current_seller, workflow: @workflow)
+    render inertia: "Workflows/Edit", props: {
+      workflow: -> { workflow_presenter.workflow_props },
+      context: -> { workflow_presenter.workflow_form_context_props },
+    }
+  end
+
+  def create
+    authorize Workflow
+
+    success, message = save_workflow
+    if success
+      redirect_to workflow_emails_path(@workflow.external_id), notice: "Workflow created successfully!"
```

Comment:
Make sure to return a [303 response here,](https://inertiajs.com/redirects) and not the default 302
```suggestion
      redirect_to workflow_emails_path(@workflow.external_id), notice: "Workflow created successfully!", status: :see_other
```